lib/Zonemaster/CLI.pm

```diff
@@ -194,7 +194,7 @@ has 'test' => (
     isa           => 'ArrayRef',
     required      => 0,
     documentation => __(
-'Specify test to run. Should be either the name of a module, or the name of a module and the name of a method in that module separated by a "/" character (Example: "Basic/basic1"). The method specified must be one that takes a zone object as its single argument. This switch can be repeated.'
+'Specify test to run. Should be either the name of a module, the name of a module and the name of a method in that module separated by a "/" character (example: "Basic/basic01") or the name of a method. The method specified must be one that takes a zone object as its single argument. This switch can be repeated.'
```

Comment:
> Is there any good reason, besides backward compatibility, to keep the basic/basic01 format?

I think backward compatibility is a good reason enough.

> The way "test" is implemented, if you specify more than one test case Zonemaster is run once per test case. Do you see that it would be possible to run all such test cases in one session? That would be an improvement.

I've looked a little bit into it, that would require some work in Zonemaster-Engine and Zonemaster-CLI. If you have an idea of the expected design, can you open an issue with it?

Reply:
The refer to the test cases as "test case" in most cases against the users. For the users it is not relevant if the test cases are implemented as methods in the code or not.

Reply:
> In most cases test cases are referred to as "test case" for the users. For the users it is not relevant if the test cases are implemented as methods in the code or not. The documentation should better be updated.

I'm not sure I understand what you mean here. Could you explain please?

Reply:
In
```
'Specify test to run. Should be either the name of a module, the name of a module and the name of a method in that module separated by a "/" character (example: "Basic/basic01") or the name of a method. The method specified must be one that takes a zone object as its single argument. This switch can be repeated.'
```
please update `the name of a method` (two places) to `the name of a test case` to be consistent with other documents.

What does `The method specified must be one that takes a zone object as its single argument` mean? Are there any test cases that do not need a single zone name? Does the user ever provide a `zone object`? Isn't that an internal unit?